plexus-compilers/plexus-compiler-eclipse/src/main/java/org/codehaus/plexus/compiler/eclipse/EclipseJavaCompiler.java

```diff
@@ -129,7 +129,10 @@ public CompilerResult performCompile( CompilerConfiguration config )
         }
         else
         {
-            StringBuilder warns = new StringBuilder();
+            String warnings = config.getWarnings();
+            StringBuilder warns = (null == warnings || warnings.isEmpty())
```

Comment:
what about `StringUtils.isEmpty`?

Reply:
I am not familiar with the Apache libraries. I will make this switch.